dev/raphael.core.js

```diff
@@ -339,7 +339,8 @@
         b = d.firstChild;
         b.style.behavior = "url(#default#VML)";
         if (!(b && typeof b.adj == "object")) {
-            return (R.type = E);
+            R.type = E;
```

Comment:
Why split `return R.type = E;` into 2 statements? I can't see why would any programmer would be confused about it!  O_o